modules/social_features/social_profile/social_profile.module

```diff
@@ -440,7 +440,7 @@ function _social_profile_check_property_access(Profile $profile, AccountProxyInt
 
       // Prepare some variables for checking on the language access.
       $global_show_lang = \Drupal::config('social_profile.settings')->get('social_profile_show_language');
-      $profile_show_lang = \Drupal::service('user.data')->get('social_profile_privacy', $profile->get('uid')->target_id, 'lang_info') ?? NULL;
+      $profile_show_lang = \Drupal::service('user.data')->get('social_profile_privacy', $profile->getFieldValue('uid', 'target_id'), 'lang_info') ?? NULL;
```

Comment:
Please remove this change from this PR. I understand it's slightly cleaner but it's unrelated to work being done here and causes a merge conflict with another work in progress PR that'll remove this altogether :D

Reply:
Sure.